phalcon/cli/router.zep

```diff
@@ -407,7 +391,7 @@ class Router implements \Phalcon\Di\InjectionAwareInterface
 	 * @param string/array paths
 	 * @return Phalcon\Cli\Router\Route
 	 */
-	public function add(string! pattern, paths = null) -> <Router>
+	public function add(string! pattern, paths = null) -> <Route>
```

Comment:
I don't want to get rid of `string|array` for the `paths` parameter. And there's a use statement mapping `Route` to `\Phalcon\Cli\Router\Route` - is this what you mean?